src/xcode/ENA/ENA/Source/HTTPClient/Resources/TeleTan/TeleTanSendModel.swift

```diff
@@ -4,7 +4,7 @@
 
 import Foundation
 
-struct KeyModel: PaddingResource {
+struct TeleTanSendModel: PaddingResource {
```

Comment:
Same question here, why is it named TeleTan? The key type can also be qrCode or guid

Reply:
Good catch! There was a renaming failure. registrationToken and teleTan got swapped. I corrected this.

Reply:
That is right, but the KeyModel was only used in combination with a TeleTanResource. So I thought i change the name to be consistent.
Otherwise, i could revert the naming here and move the model out of the folder TeleTan.
@ArturFriesen what do you think?